client/src/document/organisms/sidebar/index.tsx

```diff
@@ -28,6 +28,16 @@ export function SidebarContainer({ doc, children }) {
     }
   }, [isSidebarOpen]);
 
+  useEffect(() => {
+    const sidebar = document.querySelector(".sidebar");
```

Comment:
better to select with the ID, in case anything else gets a sidebar class in the future
```suggestion
    const sidebar = document.querySelector("#sidebar-quicklinks");
```